test/swar/BasicOperations.cpp

```diff
@@ -565,10 +583,188 @@ TEST_CASE(
-    CHECK(S4_32(0x0F0C'F000).value() == saturatingUnsignedAddition(S4_32(0x0804'F000), S4_32(0x0808'F000)).value());
+    CHECK(S4_16(0x0200).value() == saturatingUnsignedAddition(S4_16(0x0100), S4_16(0x0100)).value());
+    CHECK(S4_16(0x0400).value() == saturatingUnsignedAddition(S4_16(0x0100), S4_16(0x0300)).value());
+    CHECK(S4_16(0x0B00).value() == saturatingUnsignedAddition(S4_16(0x0800), S4_16(0x0300)).value());
+    CHECK(S4_16(0x0F00).value() == saturatingUnsignedAddition(S4_16(0x0800), S4_16(0x0700)).value());
+    CHECK(S4_16(0x0F00).value() == saturatingUnsignedAddition(S4_16(0x0800), S4_16(0x0800)).value()); 
+    CHECK(S4_32(0x0F0C'F000).value() == saturatingUnsignedAddition(S4_32(0x0804'F000), S4_32(0x0808'F000)).value()); 
+}
+
+TEST_CASE(
+    "cheapNullChecking",
+    "[swar]"
+) {
+    {
+        // The last value / lsb null lane is all these care about, but the
+        // implementation is so tied to the execution we check both.
+        CHECK(S4_32(0x0008'0080).value() == firstZeroLane<4, uint32_t>( S4_32(0xaaa0'aa0a)));
+        CHECK(1 == S4_32(firstZeroLane<4, uint32_t>( S4_32(0xaaa0'aa0a))).lsbIndex());
+        CHECK(S4_32(0x8888'8888).value() == firstZeroLane<4, uint32_t>( S4_32(0x1110'1110)));
+        CHECK(S4_32(0x8800'0000).value() == firstZeroLane<4, uint32_t>( S4_32(0x1025'3552)));
+        CHECK(S4_32(0x8880'0000).value() == firstZeroLane<4, uint32_t>( S4_32(0x1105'5352)));
+
+        // firstZeroLane finds the least significant zero (little endian). This
+        // results in the least significant zero being marked, and everything more
+        // significant than it being turned into garbage data.
+        const char * c = "aaa\0aaa\0";
+        uint64_t x;
+        memcpy(&x, c, 8);
+        S8_64 testBytes{x};
+        auto nulls = firstZeroLane<8, uint64_t>(testBytes);
+        CHECK(3 == S8_64(nulls).lsbIndex());
+    }
+
+    {
+        CHECK(S4_32(0x0000'0000).value() == firstMatchingLane<4, uint32_t>(S4_32(0x7321'4257),
+                                                                           S4_32(0x0000'0000)));
+        CHECK(S4_32(0x0000'8000).value() == firstMatchingLane<4, uint32_t>(S4_32(0x7325'0257),
+                                                                           S4_32(0x0000'0000)));
+        CHECK(S4_32(0x0000'0000).value() == firstMatchingLane<4, uint32_t>(S4_32(0x7341'4157),
+                                                                           S4_32(0x2222'2222)));
+        CHECK(S4_32(0x0880'0800).value() == firstMatchingLane<4, uint32_t>(S4_32(0x7321'4257), 
+                                                                           S4_32(0x2222'2222)));
+        CHECK(S4_32(0x0880'0800).value() == firstMatchingLane<4, uint32_t>(S4_32(0x7321'4357), 
+                                                                           S4_32(0x2222'3333)));
+    }
```

Comment:
This indentation style is not acceptable.
Do not introduce accidental indentation levels.
One indentation level means one level of detail more, and nothing more.
This indentation style moves the significant code far to the right, this is counterproductive by itself.
I treat these mistakes as fundamental challenges to the doctrine represented in this codebase.  I am literally tired of fighting this every PR review, it is deplorable that you force me to repeat myself so often about this.

Reply:
I don't care.
I changed formatting per request "The formatting of these checks require work to verify.
Perhaps use normal indentation to facilitate the comprehension?"
```
CHECK(S4_32(0x0880'0800).value() == 
    firstMatchingLane<4, uint32_t>(S4_32(0x7321'4357), 
        S4_32(0x2222'3333)));
```
Doesn't actually help comprehension (afaict), so I figured you must have meant 'line them up so matches are obvious'.
You must have some other indentation scheme in mind to assist comprehension. Please specify what it is.